### DISCUSSION TITLE
Is a Wii port Possible/Viable?

### DISCUSSION THREAD

**Discussion by PdroSantana** [nzp-team/nzportable#592] [General]
I know that you guys got a lot of version running, but the Wii could get a port, no need for motion controls, just a regular classic controller, GC controller or joycon nunchuck setup like can be found on caveEx.

Since the wii has a quake engine port already already, i thought it may be possible but not shure if it's viable for the team.

**Comment by MotoLegacy** [comment]
Maintaining *another* sourceport for development would be a major headache.

**Reply by PdroSantana** [reply]
I Understand, thanks for the awnser.

**Comment by lavenderdotpet** [comment]
it would def be possible @PdroSantana  if u have the ability to do it u could def fork it and make an fte port for wii but yeah @MotoLegacy  has enough on her plate

**Reply by MotoLegacy** [reply]
Being FTE-based isn't a requirement. You can port-forward a lot of the changes to the glQuake and Quakespasm based iterations. Porting FTE would just be making your life unnecessarily harder.
